README.md

```diff
@@ -144,23 +144,26 @@ Upgrading
 Significant parameter changes are listed below:
 
 * The PuppetDB module now supports PuppetDB 3.0.0 by default
-* If you want to use 5.x of the module with PuppetDB 2.x, you'll need to set `puppetdb_version => 2.y.z` or `terminus_package => 2.y.z`
-* The `puppetdb::master:puppetdb_conf` class has added a `$legacy_terminus` parameter which will be set the correct default if you set `puppetdb_version => 2.y.z` or `terminus_package => 2.y.z`.
-* The default `test_url` for the `PuppetDBConnValidator` has also been chaged to `/pdb/meta/v1/version` but will also be set the correct default if you set `puppetdb_version => 2.y.z` or `terminus_package => 2.y.z`.
+* If you want to use 5.x of the module with PuppetDB 2.x, you'll need to use the new `puppetdb::globals` class to set the version of PuppetDB you're using explicitly. The ability to configure the version has been therefore moved out of the `puppetdb` and `puppetdb::server` classes.
 For example if your config looked like this before:
 ~~~ruby
-class { 'puppetdb::master::config':
-      puppetdb_server => 'foo.example.com',
-      puppetdb_version => present,
+class {'puppetdb':
+  puppetdb_version => '2.3.5',
 }
+include puppetdb::master::config
 ~~~
 and you'd still like to use the module with PuppetDB 2.3.5, all you'd have to change would be:
 ~~~ruby
-class { 'puppetdb::master::config':
-      puppetdb_server => 'foo.example.com',
-      terminus_package => '2.3.5',
+class { 'puppetdb::globals':
+  version => '2.3.5',
 }
+include puppetdb
```

Comment:
Purely a fine point, and a matter of a style, but we should promote the class { } style instead of include generally, I prefer it for instruction, so its easier to explain to users how to add parameters versus the include style which requires a bigger change.